app/controllers/dashboard_controller.rb

```diff
@@ -202,13 +202,20 @@ def landing_page
   def new_landing_page
     @search_results = search_results(params[:search])
 
-    @org_owners = User.findaproject_orgs.order(:display_name).joins(:collections).left_outer_joins(:document_sets).distinct
-    @individual_owners = User.findaproject_individuals.order(:display_name).left_outer_joins(:document_sets).joins(:collections).distinct
-    
-
-    # Sampled Randomly down to 8 items for Carousel
-    docsets = DocumentSet.carousel.includes(:owner).where(owner_user_id: @org_owners.ids + @individual_owners.ids).sample(5)
-    colls = Collection.carousel.includes(:owner).where(owner_user_id: @org_owners.ids + @individual_owners.ids).sample(5)
+    users = User.owners
+                .joins(:collections)
+                .left_outer_joins(:document_sets)
+                .includes({ random_collections: :tags }, { random_document_sets: { collection: :tags } })
```

Comment:
This is a bit tricky. So we cannot preload queries with dynamic like random.

So what I did here is remove the random_sample scope in random_collections, then we sample later via array to SQL calls.

Reply:
Although, in hindsight, this makes 'random_collection' and 'random_document_sets' scope feel like incorrect labels now.

Reply:
What does `collection: :tags` do here?  I'm a little confused about whether this changes functionality.

We'd be fine with some more predictable/cacheable pseudo-random behavior, like selecting collections whose `id` ends with the same last digit as the current day-of-year.